locations/google_url.py

```diff
@@ -94,7 +94,10 @@ def get_query_param(link, query_param):
 
     if "/maps.google.com/" in url:
         for ll in get_query_param(url, "ll"):
-            lat, lon = ll.split(",")
+            lat_lon = ll.split(",")
+            if len(lat_lon) == 3:  # Zoom value can be included in ll options
```

Comment:
There's an existing test suite for google_url. Can you add a case for this please?

Reply:
Sure, have done with one such url from one of these pages